### ISSUE TITLE
Build problem

### ISSUE THREAD

**Issue by UshakovVasilii** [aichaos/rivescript-java#25]
```
$ ./gradlew build
:rivescript-core:compileJava UP-TO-DATE
:rivescript-core:processResources UP-TO-DATE
:rivescript-core:classes UP-TO-DATE
:rivescript-core:jar UP-TO-DATE
:api
:docs
:docsZip
:rivescript-core:javadoc UP-TO-DATE
:rivescript-core:javadocJar UP-TO-DATE
:rivescript-core:sourcesJar UP-TO-DATE
:distZip
:assemble
:compileJava UP-TO-DATE
:processResources UP-TO-DATE
:classes UP-TO-DATE
:compileTestJava UP-TO-DATE
:processTestResources UP-TO-DATE
:testClasses UP-TO-DATE
:test UP-TO-DATE
:check UP-TO-DATE
:build
:rivescript-core:signArchives
FAILURE: Build failed with an exception.

* What went wrong:
java.lang.NullPointerException (no error message)

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.


BUILD FAILED

Total time: 4.414 secs
```
[gradle_stacktrace.txt](https://github.com/aichaos/rivescript-java/files/573136/gradle_stacktrace.txt)

**Comment by kirsle** [comment]
@marceloverdijk :arrow_up:

**Comment by marceloverdijk** [comment]
@kirsle this should be gone after PR #24 is merged.

https://github.com/aichaos/rivescript-java/pull/24/files#diff-c197962302397baf3a4cc36463dce5ea contains the `required { gradle.taskGraph.hasTask("uploadArchives") }` to avoid signing when just building locally.

**Comment by marceloverdijk** [comment]
If it might take some time to merge #24 then we add that line to `build.gradle` in `master` as well.

**Comment by marceloverdijk** [comment]
Solved after merging #24.
